incubator/neo4j/templates/statefulset.yaml

```diff
@@ -98,9 +100,9 @@ spec:
   - metadata:
       name: datadir
       annotations:
-        volume.alpha.kubernetes.io/storage-class: "{{.Values.StorageClass}}"
+        volume.alpha.kubernetes.io/storage-class: "{{.Values.storageClass}}"
```

Comment:
You only support alpha. You should also support beta. Here's an example for doing this:

https://github.com/kubernetes/charts/blob/master/stable/mongodb-replicaset/templates/mongodb-statefulset.yaml#L175-L199

To be honest, I wouldn't even support alpha any more.